docs/usage/validation_errors.md

```diff
@@ -1494,13 +1516,59 @@ class Model(BaseModel):
 
 try:
     Model(x=None)
-except ValidationError as exc_info:
-    assert exc_info.errors()[0]['type'] == 'time_type'
+except ValidationError as exc:
+    print(repr(exc.errors()[0]['type']))
+    #> 'time_type'
+```
+
+This error is also raised for strict fields when the input value is not an instance of `time`.
+
+## `timezone_aware`
+
+This error is raised when the `datetime` value provided for a timezone-aware `datetime` field
+doesn't have timezone information:
+
+```py
+from datetime import datetime
+
+from pydantic import AwareDatetime, BaseModel, ValidationError
+
+
+class Model(BaseModel):
+    x: AwareDatetime
+
+
+try:
+    Model(x=datetime.now())
+except ValidationError as exc:
+    print(repr(exc.errors()[0]['type']))
+    #> 'timezone_aware'
+```
+
+## `timezone_naive`
+
+This error is raised when the `datetime` value provided for a timezone-naive `datetime` field
+has timezone info:
+
+```py
+from datetime import datetime, timezone
+
+from pydantic import BaseModel, NaiveDatetime, ValidationError
+
+
+class Model(BaseModel):
+    x: NaiveDatetime
+
+
+try:
+    Model(x=datetime.now(tz=timezone.utc))
+except ValidationError as exc:
+    print(repr(exc.errors()[0]['type']))
```

Comment:
same here. print statement output is missed

Reply:
thanks